networking_cisco/plugins/cisco/l3/drivers/asr1k/aci_asr1k_routertype_driver.py

```diff
@@ -149,8 +149,9 @@ def unschedule_router_postcommit(self, context, router_context):
             context, router_context)
 
     def add_router_interface_precommit(self, context, r_port_context):
-        super(AciASR1kL3RouterDriver, self).add_router_interface_precommit(
-            context, r_port_context)
```

Comment:
It doesn't. Here's the superclass method:
https://github.com/noironetworks/networking-cisco/blob/master-branch/networking_cisco/plugins/cisco/l3/drivers/asr1k/asr1k_routertype_driver.py#L124-L151

Note that all it does is make this check.